.changeset/gentle-yaks-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": minor
```

Comment:
Based off other CSS modules PRs in this repo (example: https://github.com/primer/react/pull/5271/files#diff-b5b0f0cbdba0850831f60c159e84b5bbd32224d3a9dcfccea7e3443cca64520aR2)

Reply:
Minor is what we usually go for when things are first converted